mama/c_cpp/src/c/bridge.c

```diff
@@ -209,3 +239,16 @@ mamaBridgeImpl_areEntitlementsDeferred (mamaBridge bridgeImpl)
     }
     return 0;
 }
+
+/* This must be called immediately after bridge init function called */
```

Comment:
is this done at the mama level or do we need to ad it to the bridge developer documentation?